modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java

```diff
@@ -138,6 +139,34 @@ public ObjectEntry fetchObjectEntry(long objectEntryId)
 		return objectEntry;
 	}
 
+	public List<ObjectEntry> getManyToManyObjectEntries(
+			long groupId, long objectRelationshipId, long primaryKey,
+			boolean related, boolean reverse, int start, int end)
+		throws PortalException {
+
+		List<ObjectEntry> objectEntries =
+			objectEntryLocalService.getManyToManyObjectEntries(
+				groupId, objectRelationshipId, primaryKey, related, reverse,
+				start, end);
+
+		for (ObjectEntry objectEntry : objectEntries) {
+			_checkModelResourcePermission(
+				objectEntry.getObjectDefinitionId(),
+				objectEntry.getObjectEntryId(), ActionKeys.VIEW);
+		}
+
+		return objectEntries;
+	}
+
+	public int getManyToManyObjectEntriesCount(
```

Comment:
@marcelabc I don't understand why these new methods don't have the `@Override`  😕

Reply:
Sorry, I forgot :+1: